### ISSUE TITLE
Does SC 3.2.3 Consistent Navigation allow for links to be moved from one navigation group to another?

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#1169]
During our discussion of the proposed Finding Help criterion the comment came up that SC 3.2.3 as written applies to links within a navigation block and thus moving a navigation link from one navigation block to another navigation block would actually not be a violation.  If this is indeed the case then it might be worth clarifying this in the understanding document for 3.2.3 for people as it's a subtle distinction that many people would miss.

**Comment by alastc** [comment]
Hi Jon, did you mean "as written **does not** applies to links within a navigation block"?

**Comment by mraccess77** [comment]
No, I think the discussion is that it does apply to links within a group.
